server/controllers/sold.js

```diff
@@ -11,25 +11,47 @@
 /* eslint-disable import/no-unresolved */
```

Comment:
You have a lot commented out eslint errors, whats up?

Reply:
yes I will work on that too